driver_cpl/cime_config/testdefs/testlist_drv.xml

```diff
@@ -1,316 +1,283 @@
 <?xml version="1.0"?>
 <testlist version="2.0">
-  <test name="CME_Ld3" grid="f45_g37_rx1" compset="A">
```

Comment:
In the course of rewriting the tests I found that most cime tests could be much shorter and a couple (SEQ and NCK) could be modified a little to be more effective.